lib/core/src/lib/pipes/full-name-email-required.token.ts

```diff
@@ -0,0 +1,23 @@
+/*!
+ * @license
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable rxjs/no-subject-value */
+/* eslint-disable @typescript-eslint/naming-convention */
+
+import { InjectionToken } from '@angular/core';
+
+export const ADF_FULL_NAME_PIPE_INCLUDE_EMAIL = new InjectionToken<boolean>('ADF_FULL_NAME_PIPE_INCLUDE_EMAIL');
```

Comment:
I would say this is too over-engineered way to format a user full name... you literally introduce tone of code and tests just to compose "first name" and "last name". should not the applications or components just do that in the code instead?

Reply:
The problem here is about the requirement requested by the POs.

They want to display also the email in the adf-people component that uses this pipe. But they want it to be displayed in HxP but not in APA. 

This is the less intrusive way I could think on, because the other alternative would be create a custom people component in hxp-apps which is exactly the same as the adf-people but with a custom pipe but anyway the pipe must be like this as it must be used differently in APA and in HxP

Reply:
ok, could probably work for now, but this pipe looks too over-engineered already, we can completely get rid of this pipe in the future releases probably